pages/contact.js

```diff
@@ -0,0 +1,66 @@
+import React, {Component} from 'react';
+import NavbarBootstrap from '../components/NavbarBootstrap';
+import Card from 'react-bootstrap/Card';
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import Container from 'react-bootstrap/Container';
+
+class Contact extends Component {
+    render() {
+        return (<div>
```

Comment:
newline here after `(`
```suggestion
        return (
            <div>
            ....
        );
```

Reply:
Should I also make a new line where return is closed?

Currently:
```
    </div>)
```
Should I:
```
    </div>
)
```

Reply:
fixed